src/util/macro_util.rs

```diff
@@ -359,6 +359,31 @@ macro_rules! struct_has_padding {
     };
 }
 
+/// Does the struct type `$t` have padding?
+///
+/// `$ts` is the list of the type of every field in `$t`. `$t` must be a
+/// `repr(C)` struct type, or else `struct_has_padding!`'s result may be
+/// meaningless.
+#[doc(hidden)] // `#[macro_export]` bypasses this module's `#[doc(hidden)]`.
+#[macro_export]
+macro_rules! cstruct_has_padding {
```

Comment:
```suggestion
macro_rules! repr_c_struct_has_padding {
```

I found `cstruct` to be confusing until I saw `repr(C)` in the doc comment.